database/seeders/StudentSeeder.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
this file was generated using the command

`php artisan make:seeder StudentSeeder`